packages/core/hooks/useOpenWithMenuItems/index.tsx

```diff
@@ -240,7 +275,24 @@ export default (fileDetails?: FileDetail, filters?: FileFilter[]): IContextualMe
     const annotationNameToAnnotationMap = useSelector(
         metadata.selectors.getAnnotationNameToAnnotationMap
     );
+    const annotations = useSelector(metadata.selectors.getAnnotations);
```

Comment:
Minor: if you're only using the annotations to get the names in `useOpenInCfe`, could theoretically use `Object.keys` on the `annotationNameToAnnotationMap` selector above instead